HexFiend_2.xcodeproj/xcshareddata/xcschemes/Debug.xcscheme

```diff
@@ -15,7 +15,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "8D1107260486CEB800E47090"
-               BuildableName = "Hex Fiend.app"
+               BuildableName = "MYGH Hex Fiend.app"
```

Comment:
please revert

Reply:
Apologies. I am new to the GitHub collaborative environment. I'm trying to be a good citizen. Also, using GutHub Desktop I am seeing the same mess-up in Release.xcscheme as well. I think I have fixed both. GitHub Desktop is also noting changes in Debug.xcscheme.swp. I'm not sure how to proceed to do to fix that. Again, Apologies.

Reply:
I didn't realize at the time this PR was created from your master branch. See my comments below.